tRecorderApi/media/dump/1498678675.09d140d7ea-4bbc-49b2-a840-ace06c1a5da6/test.txt

```diff
@@ -0,0 +1 @@
+Testing
```

Comment:
gitignore the dump folder to prevent having a ton of these in your commit, please 👍